tests/smoke_test/test_smoke_private_endpoints.py

```diff
@@ -0,0 +1,191 @@
+        ]
+    )
+    def test_odw_datalake_private_endpoints(self, endpoint_name: str):
+        all_endpoints = self.get_all_endpoints(StoragePrivateEndpointManager, f"pins-rg-data-odw-{self.ENV}-uks", self.DATA_LAKE_STORAGE)
+        self.validate_private_endpoint(all_endpoints, endpoint_name)
+
+    @pytest.mark.parametrize(
+        "endpoint_name",
+        [
+            f"pins-pe-pinskvsynwodw{ENV}uks",
+            f"pins-pe-pinskvsynwodw{ENV}uks-tooling"
+        ]
+    )
+    def test_odw_keyvault_private_endpoints(self, endpoint_name: str):
+        all_endpoints = self.get_all_endpoints(KeyVaultPrivateEndpointManager, f"pins-rg-data-odw-{self.ENV}-uks", f"pinskvsynwodw{self.ENV}uks")
+        self.validate_private_endpoint(all_endpoints, endpoint_name)
+
+    @pytest.mark.parametrize(
+        "endpoint_name",
+        [
+            f"pins-pe-pinskvmgmtodw{ENV}uks"
+        ]
+    )
+    def test_management_keyvault_private_endpoints(self, endpoint_name: str):
+        all_endpoints = self.get_all_endpoints(KeyVaultPrivateEndpointManager, f"pins-rg-datamgmt-odw-{self.ENV}-uks", f"pinskvmgmtodw{self.ENV}uks")
+        self.validate_private_endpoint(all_endpoints, endpoint_name)
+
+    @pytest.mark.parametrize(
+        "endpoint_name",
+        [
+            f"pins-pe-syn-devops-odw-{ENV}-uks",
+            f"pins-pe-syn-ssql-odw-{ENV}-uks",
+            f"pins-pe-syn-devops-tooling-odw-{ENV}-uks",
+        ]
+    )
+    def test_odw_synapse_private_endpoints(self, endpoint_name: str):
+        all_endpoints = self.get_all_endpoints(SynapsePrivateEndpointManager, f"pins-rg-data-odw-{self.ENV}-uks", f"pins-synw-odw-{self.ENV}-uks")
+        self.validate_private_endpoint(all_endpoints, endpoint_name)
+
+    @pytest.mark.parametrize(
+        "endpoint_name",
+        [
+            f"pins-pe-syn-ws-odw-{ENV}-uks",
+            f"pins-pe-syn-ws-tooling-odw-{ENV}-uks"
+        ]
+    )
+    def test_odw_synapse_private_endpoints(self, endpoint_name: str):
+        all_endpoints = SynapsePrivateLinkHubsPrivateEndpointManager(
+            f"pinsplsynwsodw{self.ENV}uks",
+            self.SUBSCRIPTION_ID,
+            f"pins-rg-network-odw-{self.ENV}-uks"
+        ).get_all()
+        self.validate_private_endpoint(all_endpoints, endpoint_name)
+
+    @pytest.mark.parametrize(
+        "endpoint_name",
+        [
+            f"synapse-st-dfs--{DATA_LAKE_STORAGE}",
+            f"synapse-st-dfs--{DATA_LAKE_FAILOVER_STORAGE}",
+            f"synapse-mpe-kv--odw-{ENV}-uks"
+        ]
+    )
+    def test_odw_synapse_managed_private_endpoints(self, endpoint_name: str):
+        all_endpoints = SynapseManagedPrivateEndpointManager(f"pins-synw-odw-{self.ENV}-uks").get_all()
+        self.validate_managed_private_endpoint(all_endpoints, endpoint_name)
+
+    @pytest.mark.parametrize(
+        "endpoint_name",
+        [
+            #f"synapse-mpe-appeals-bo--odw-{ENV}-uks",
+            f"synapse-mpe-purview-account--odw-{ENV}-uks",
```

Comment:
You could add pytest.param(...., id=.....) for each parametrize to make logs more readable when the test fail maybe ?